osm2pgsql.lua

```diff
@@ -474,6 +492,129 @@ function osm2pgsql.process_node(object)
     object.tags.wikidata == 'Q227569' or object.tags.wikidata == 'Q62172') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
```

Comment:
Greece surprises me here.

Reply:
Thats how its marked in NE

Reply:
I stand corrected! Looks like I need to research about Spain, as well:

- https://www.euractiv.com/section/politics/short_news/greece-will-not-change-position-on-kosovo/